### ISSUE TITLE
typo?

### ISSUE THREAD

**Issue by NeveIsa** [kvfrans/splus#5]
# Training step
optimizer.train() # New in SPlus: Include this!
model.train()
loss = model(batch)
loss.backwards()
optimizer.step()
optimizer.zero_grad()

You meant "backward" and not "backwards" right?
